### ISSUE TITLE
a few pep8 details in modules

### ISSUE THREAD

**Issue by fchapoton** [sagemath/sage#34930]
<div id="comment:0"></div>

mostly about spaces

Component: **refactoring**

Author: **Frédéric Chapoton**

Branch/Commit: **[u/chapoton/34930](https://github.com/sagemath/sagetrac-mirror/tree/u/chapoton/34930) @ [`705606a`](https://github.com/sagemath/sagetrac-mirror/commit/705606adcacedba25fa5e433a093d473d3674b31)**

Reviewer: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34930_

**Comment by fchapoton** [comment]
Commit: **[`d059888`](https://github.com/sagemath/sagetrac-mirror/commit/d059888b9d5a3675a114504266a5b5fa1bb80e90)**

**Comment by fchapoton** [comment]
Branch: **[u/chapoton/34930](https://github.com/sagemath/sagetrac-mirror/tree/u/chapoton/34930)**

**Comment by fchapoton** [comment]
<div id="comment:1"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/d059888b9d5a3675a114504266a5b5fa1bb80e90"><code>d059888</code></a></td><td><code>a few pep8 details in modules/</code></td></tr></table>

**Comment by mkoeppe** [comment]
<div id="comment:2" align="right">comment:2</div>


```
sage -t --random-seed=178536346284821476134716579559621760345 sage/modules/filtered_vector_space.py  # 28 doctests failed
sage -t --random-seed=178536346284821476134716579559621760345 sage/modules/multi_filtered_vector_space.py  # 40 doctests failed
sage -t --random-seed=178536346284821476134716579559621760345 sage/schemes/toric/sheaf/constructor.py  # 3 doctests failed
sage -t --random-seed=178536346284821476134716579559621760345 sage/schemes/toric/sheaf/klyachko.py  # 21 doctests failed
```

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
<div id="comment:3"></div>

Branch pushed to git repo; I updated commit sha1. New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/705606adcacedba25fa5e433a093d473d3674b31"><code>705606a</code></a></td><td><code>fixing code in filtered vector space</code></td></tr></table>

**Comment by 7ed8c4ca-6d56-4ae9-953a-41e42b4ed313** [comment]
Changed commit from **[`d059888`](https://github.com/sagemath/sagetrac-mirror/commit/d059888b9d5a3675a114504266a5b5fa1bb80e90)** to **[`705606a`](https://github.com/sagemath/sagetrac-mirror/commit/705606adcacedba25fa5e433a093d473d3674b31)**

**Comment by fchapoton** [comment]
<div id="comment:4" align="right">comment:4</div>

fixed now

**Comment by mkoeppe** [comment]
Reviewer: **Matthias Koeppe**

**Comment by mkoeppe** [comment]
Merged in 10.0.beta0
